apps/backoffice-v2/src/common/components/molecules/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { CalendarIcon } from '@radix-ui/react-icons';
+import { formatDate } from '@/common/utils/format-date';
+import { DateRange } from 'react-day-picker';
+import { addMonths } from 'date-fns';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { Button } from '../../atoms/Button/Button';
+import { Calendar } from '../../organisms/Calendar/Calendar';
+import { Popover, PopoverContent, PopoverTrigger } from '@ballerine/ui';
+
+type DateRangePickerProps = {
+  onChange: (range: { start: Date | null; end: Date | null }) => void;
+  value: { start: Date | null; end: Date | null };
+};
+
+export function DateRangePicker(
+  { onChange, value }: DateRangePickerProps,
+  { className }: React.HTMLAttributes<HTMLDivElement>,
+) {
+  const currentDate = new Date();
+  const [date, setDate] = useState<DateRange | undefined>({
+    from: new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate()),
+    to: addMonths(
+      new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate(), 1),
+      1,
+    ),
+  });
+  const prevDateRef = useRef<DateRange | undefined>(date);
+
+  useEffect(() => {
+    if (date !== prevDateRef.current) {
+      onChange({
+        start: date?.from || null,
+        end: date?.to || null,
+      });
+      prevDateRef.current = date;
+    }
+  }, [date, onChange]);
+
+  return (
+    <div className={ctw('grid gap-2', className)}>
+      <Popover>
+        <PopoverTrigger asChild>
+          <Button
+            id="date"
+            variant={'outline'}
+            className={ctw(
+              'w-[300px] justify-start text-left font-normal',
+              !date && 'text-muted-foreground',
```

Comment:
This syntax is part of `ctw`'s and `clsx`'s advantages. 🙂 
```suggestion
             { 'text-muted-foreground': !date },
```